populate-metadata/src/atlasConnector.ts

```diff
@@ -0,0 +1,51 @@
+import * as mongodb from 'mongodb';
+import { getEnvVars } from './assertEnvVars';
+import type { DocsetsDocument, ReposBranchesDocument } from './types';
+
+const ENV_VARS = getEnvVars();
+
+let clusterZeroClient: mongodb.MongoClient;
+
+export const teardown = async (client: mongodb.MongoClient) => {
+  await client.close();
+};
+
+// Handles memoization of db object, and initial connection logic if needs to be initialized
+export const dbClient = async (uri: string) => {
+  const client = new mongodb.MongoClient(uri);
+  try {
+    await client.connect();
+    return client;
+  } catch (error) {
+    const err = `Error at client connection: ${error} `;
+    console.error(err);
+    throw err;
+  }
+};
+
+export const getPoolDb = async () => {
+  if (clusterZeroClient) {
```

Comment:
Better not to make a blank if just for else
```suggestion
  if (!clusterZeroClient) {
```